beachball.config.js

```diff
@@ -1,2 +1,2 @@
-require("./build/releasing/ts-node-register");
-module.exports = require("./build/releasing/index");
+/*require("./build/releasing/ts-node-register");
```

Comment:
should this be deleted?

Reply:
let not deleted. I will work beachball deployment later on